XCUITests/AuthenticationTest.swift

```diff
@@ -105,34 +105,58 @@ class AuthenticationTest: BaseTestCase {
       
         enablePasscode("1337")
         navigator.goto(LoginsSettings)
-        waitforExistence(app.staticTexts["Enter Passcode"])
+        
+        if #available(iOS 11, *) {
```

Comment:
I thought about this, but since master is only running on iOS 11, let's remove this kind of if statement, and only put the waitforExistence call that works on iOS 11.